foreman/foreman.spec

```diff
@@ -577,6 +618,9 @@ sed -i 's/:locations_enabled: false/:locations_enabled: true/' config/settings.y
 sed -i 's/:organizations_enabled: false/:organizations_enabled: true/' config/settings.yaml
 export BUNDLER_EXT_NOSTRICT=1
 export BUNDLER_EXT_GROUPS="default assets"
+export NODE_PATH=%{nodejs_sitelib}
```

Comment:
Sorry, this doesn't seem to work, webpack is unable to find modules:

<pre>
+ export NODE_PATH=/usr/lib/node_modules
+ NODE_PATH=/usr/lib/node_modules
+ export NODE_ENV=production
+ NODE_ENV=production
+ webpack.js --bail --config config/webpack.config.js
EntryModuleNotFoundError: Entry module not found: Error: Cannot resolve module 'babel-loader' in /builddir/build/BUILD/foreman-1.13.0
</pre>

Going back to nearly the original code, this does work:

<pre>
ln -s %{nodejs_sitelib} node_modules
</pre>